apps/opik-backend/src/main/resources/liquibase/db-app-analytics/migrations/000005_add_prompt_version_to_experiments.sql

```diff
@@ -0,0 +1,7 @@
+--liquibase formatted sql
+--changeset andrescrz:add_prompt_version_to_experiments
```

Comment:
I guess your username should appear here

Reply:
Good catch, thank you!